simulator/src/main/java/com/hazelcast/simulator/worker/metronome/MetronomeBuilder.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2008-2016, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hazelcast.simulator.worker.metronome;
+
+import static com.hazelcast.simulator.utils.Preconditions.checkNotNull;
+import static java.lang.Math.round;
+import static java.util.concurrent.TimeUnit.MICROSECONDS;
+import static java.util.concurrent.TimeUnit.MILLISECONDS;
+import static java.util.concurrent.TimeUnit.SECONDS;
+
+public class MetronomeBuilder {
+
+    private MetronomeType type = MetronomeType.BUSY_SPINNING;
```

Comment:
Shouldn't we have the same defaults in `PropertyBinding` and `MetronomeBuilder` so it's less confusing which ones are used?